Assets/Scripts/Core/GameManager.cs

```diff
@@ -51,8 +51,7 @@ public enum PlayerType { Human, AI }
 			gameMoves = new List<Move> ();
 			board = new Board ();
 			searchBoard = new Board ();
-			aiSettings.diagnostics = new Search.SearchDiagnostics ();
```

Comment:
With this change, `aiSettings.diagnostics` will not be reset to a new `Search.SearchDiagnostics` every time that `Start` is called.  Is this the desired behavior change?

Secondary question: Why move this line to the single parameter `NewGame` method rather than the two-parameter `NewGame` method?

Reply:
Hey Brandon, sorry I just quickly wrote the change for the issue referenced for help. Unfortunately I do not have the capacity to proceed or intend to follow through. 

Do you want to pick it up instead?  Let me know I can discard this one when you're done (it's up just for reference).